regression-test/common/helper.groovy

```diff
@@ -67,8 +67,8 @@ class Helper {
         return gson.toJson(body)
     }
 
-    void ccrJobDelete(table = "") {
```

Comment:
For ccr tasks, it should be able to specify the table and db to perform the operation e.g. create pause